lib/tools/aggregation.py

```diff
@@ -48,20 +48,31 @@
 # Show the environment
 armbian_utils.show_incoming_environment()
 
+COMMON_FIXED_DESKTOP_ENVIRONMENT_CONFIG_NAME = "config_base"
+
 util.SELECTED_CONFIGURATION = armbian_utils.get_from_env_or_bomb("SELECTED_CONFIGURATION")  # "cli_standard"
 util.DESKTOP_APPGROUPS_SELECTED = armbian_utils.parse_env_for_tokens("DESKTOP_APPGROUPS_SELECTED")  # ["browsers", "chat"]
 util.SRC = armbian_build_directory
 
+# All the search roots, including ARCH-varying ones.
 util.AGGREGATION_SEARCH_ROOT_ABSOLUTE_DIRS = [
 	f"{armbian_build_directory}/config",
 	f"{armbian_build_directory}/config/optional/_any_board/_config",
 	f"{armbian_build_directory}/config/optional/architectures/{ARCH}/_config",
 	f"{USERPATCHES_PATH}"
 ]
 
+# Only the common, non-arch-varying search roots.
+util.AGGREGATION_SEARCH_ROOT_ABSOLUTE_DIRS_COMMON = [
+	f"{armbian_build_directory}/config",
+	f"{armbian_build_directory}/config/optional/_any_board/_config",
```

Comment:
Trying to figure out this case ... If I understand this correctly, app groups are ignored in this package. But we would like that our default desktop contains app groups: editor, remote_desktop, chat and perhaps office. But also clean desktop is desirable ... how to handle this?

Reply:
Sincerely there's no good way to handle this in a generic desktop package.
Appgroups should be ignored for the generic package, and appgroup specific packages could be added later.

Reply:
I did few experiments later and if I understand this change correctly it builds image the same as before (if some appgroups are added it includes them) but the desktop package contains only base desktop files.

Reply:
Yes, the packages installed in image due to appgroups are not affected by this, but the scripts ran when packages are installed are affected. See specifically https://github.com/armbian/build/issues/5439#issuecomment-1621669297